products/idn/docs/identity-now/transforms/operations/lookup.md

```diff
@@ -71,9 +72,9 @@ transform involves a `table` entry of key-value pairs:
 
 ## Examples
 
-This transform tries to map a telephone area code to a city in Texas. There is
-no default entry in the table map, so the transform returns null if there is no
-provided area code that is not one of the provided four values.
+This transform tries to map a telephone area code to a city in Texas. If there
+is no provided area code in one of the provided four values, the
+transform will return the default value of `Unknown Area".
 
```

Comment:
Can you change this to "If there is no area code in the four provided values, the transform will return the default value, "Unknown Area."